install/ci-vm/ci-windows/ci/runCI.bat

```diff
@@ -25,12 +25,13 @@ if EXIST "%dstDir%\ccextractorwinfull.exe" (
     call :executeCommand cd %suiteDstDir%
     call :executeCommand "%tester%" --entries "%testFile%" --executable "ccextractorwinfull.exe" --tempfolder "%tempFolder%" --timeout 3000 --reportfolder "%reportFolder%" --resultfolder "%resultFolder%" --samplefolder "%sampleFolder%" --method Server --url "%reportURL%"
 
-    curl -s -A "%userAgent%" --form "type=logupload" --form "file=@%logFile%" -w "\n" "%reportURL%"
+    curl -s -A "%userAgent%" --form "type=logupload" --form "file=@%logFile%" -w "\n" "%reportURL%" >> "%logFile%"
     timeout 10
 
     echo Done running tests
     call :postStatus "completed" "Ran all tests"
 
+    timeout 60
```

Comment:
A timeout of 60, may I know the reason?

Reply:
I tested this before making changes in `controllers.py`. I would test with lower time and make changes if required.

Reply:
Timeout is already present in `postStatus` function, hence removed this timeout.